app/plot_app/vtol_tailsitter.py

```diff
@@ -0,0 +1,81 @@
+# VTOL tailsitter attitude and rate correction code
+# tailsitter uses 90 degree rotation in pitch which is hardcoded in rather than consistently reported in estimated and setpoint
+# use setpoint values as ground truth here and correct estimated by 90 degrees
+
+from scipy.spatial.transform import Rotation as Rot
+import numpy as np
+
+def correct_tailsitter_attitude_and_rates(data,vtol_states):
+
+    for d in data:
+        if d.name == 'vehicle_attitude':
```

Comment:
Can you add some flags to check if the topics were found?

Reply:
made this consistent with other areas where variables are referenced - let me know if you want an explicit flag of some sort set